doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst

```diff
@@ -55,13 +55,19 @@ More detailed information on MTC can be found in the :doc:`MoveIt Task Construct
 -----------------
 If you haven't already done so, make sure you've completed the steps in :doc:`Getting Started </doc/tutorials/getting_started/getting_started>`.
 
-2.1 Download MoveIt Task Constructor
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Move into your colcon workspace and pull the MoveIt Task Constructor source: ::
+Move into your colcon workspace and pull the MoveIt Task Constructor source, where ``<branch>`` can be e.g. ``humble`` for ROS Humble, or ``ros2`` for the latest version compatible with MoveIt 2 ``main``: ::
```

Comment:
Did you intend to delete this `2.1` heading?

Reply:
@sea-bass Yes, I removed the this heading from section 2 because it did not include multiple subheadings. Do you suggest adding it?

Reply:
No, was just checking!